lib/repl.js

```diff
@@ -47,15 +47,16 @@ const {
   makeRequireFunction,
   addBuiltinLibsToObject
 } = require('internal/modules/cjs/helpers');
-const internalUtil = require('internal/util');
```

Comment:
If I am not mistaken it will not (easily) be possible to lazy load `internal/util`. I tried to do that and it is used for a lot of things that are eagerly loaded.